stable/artifactory/templates/ingress.yaml

```diff
@@ -15,9 +15,12 @@ metadata:
       {{ $key }}: {{ $value | quote }}
     {{- end }}
 spec:
+  {{- if .Values.ingress.defaultBackend.enabled }}
```

Comment:
Please add this to `artifactory-ha` as well. We like to keep them aligned where possible.

Reply:
Can do, Altho I don't have artifactory-ha running on my cluster right now so I cannot fully test it.

Also, sorry about the README.md, I had already modified it but left the file out of the commit. :/

Reply:
done in 9e2a15175f76af2e5baba908b28f9d07329e4ffa